spec/awful/placement_spec.lua

```diff
@@ -0,0 +1,28 @@
+describe("awful.placement", function()
+    -- tiny hack to make sure we don't get errors about connect_signal
+    package.loaded["awful.screen"] = {}
+
+    local place = require("awful.placement")
+
+    it("awful.placement.closest_corner (top left)", function()
+        local _, corner = place.closest_corner(
+        {coords=function() return {x = 100, y=100} end},
+        {include_sides = true, bounding_rect = {x=0, y=0, width=200, height=200}}
+        )
+        assert(corner == "top_left")
```

Comment:
Eventually, if you add more, favor the likes of `assert.are_equal` which are part of the testing framework. It makes prettier output.

Reply:
> Eventually, if you add more, favor the likes of `assert.are_equal` which are part of the testing framework. It makes prettier output.

Gotcha, will keep in mind for the future.